src/api/firebase.js

```diff
@@ -29,6 +29,54 @@ export function streamListItems(listId, handleSuccess) {
 	return onSnapshot(listCollectionRef, handleSuccess);
 }
 
+/**
+ * comparePurchaseUrgency CURRENT FUNCTIONS:
+ * - sort for inactive items last, then
+ * - sort for overdue items to be listed at the top that aren't inactive, then
+ * - sort to most urgent date next purchased, then
+ * - sort to earliest alphabet name if date next purchased is the same
+ */
+export function comparePurchaseUrgency(item1, item2) {
+	//Declare current date
+	const currentDate = new Date();
+
+	//Returns the days until next purchase between currentDate and dateNextPurchased for item argument
+	function getDaysUntilNextPurchase(item) {
+		return getDaysBetweenDates(currentDate, item.dateNextPurchased.toDate());
+	}
+
+	//Inactive item conditional
+	if (getDaysUntilNextPurchase(item1) >= 60) {
```

Comment:
Curious - in office hours we talked about maybe refactoring this into a `switch` statement. Any thoughts on that?

Reply:
Office hours was incredibly helpful! When playing around with switch values it got really tricky since we were using imprecise values and multiple condition variations both in the back and front end. It made more sense at the time to continue with if-else since we were using all imprecise values (with greater and and less than attributes) and not precise values as usually recommended with switch.